include/openssl/ec.h

```diff
@@ -1200,8 +1204,12 @@ void ECDSA_SIG_free(ECDSA_SIG *sig);
  *  \param  pp   pointer to a unsigned char pointer for the output or NULL
  *  \return the length of the DER encoded ECDSA_SIG object or a negative value
  *          on error
+ *
+ * Originally these were macro exapanded from:
+ *  DECLARE_ASN1_ENCODE_FUNCTIONS_only(ECDSA_SIG, ECDSA_SIG)
```

Comment:
Why do we make this change?

Reply:
Unless I introduce a new macro, these functions can't be deprecated otherwise.
Not deprecating them is an option -- do we want this?

Reply:
Well you're not deprecating them anymore anyway!

Reply:
Convention has been to use ```|foo|``` to indicate parameters, and just run the comment text as a set of prose.  Note to use javadoc conventions.  (For example, remove the : and newline if it doesn't wrap from 1208/1209.)

Reply:
Reminder to self: don't try to think late on a Friday evening after an hectic week.

Reply:
Reverted this to its original form.

Reworking the comments to follow the parameter convention is for another PR.